src/class-permissions.php

```diff
@@ -83,6 +83,31 @@ public static function current_user_can_use_pch_feature(
 		$current_user = wp_get_current_user();
 		$user_roles   = $current_user->roles;
 
+		/**
+		 * Filters whether the current user can use the specified Content Helper
+		 * feature.
+		 *
+		 * This filter can be used to override the default permissions check.
+		 *
+		 * @since 3.16.2
+		 *
+		 * @param bool   $current_user_can_use_pch_feature Whether the current user can use the feature.
+		 * @param string $feature_name The feature's name.
+		 * @param \WP_User $current_user The current user object.
+		 * @param int|false $post_id The post ID, if the check is for a specific post.
+		 */
+		$filtered_current_user_can_use_pch_feature = apply_filters(
+			'wp_parsely_current_user_can_use_pch_feature',
+			false,
+			$feature_name,
+			$current_user,
+			$post_id
+		);
+
+		if ( $filtered_current_user_can_use_pch_feature ) {
```

Comment:
```suggestion
		if ( true === $filtered_current_user_can_use_pch_feature ) {
```
What do you think about that? I think that being more restrictive and explicit here makes sense, since filter consumers could return non-boolean results.

Reply:
Yes, that makes sense, thank you for the suggestion!